e2e/express/package.json

```diff
@@ -0,0 +1,17 @@
+{
+	"name": "e2e-express",
```

Comment:
@deltaepsilon the reason i'm naming this `e2e-express` is that `express` collides with the `express` js sdk itself.